### DISCUSSION TITLE
Always crashing on windows 11

### DISCUSSION THREAD

**Discussion by SKaterTux** [KRTirtho/spotube#736] [Q&A]
Spotube did work fine but starting about a week ago it just crashes every single time, the new update didn't fix this either

**Comment by KRTirtho** [comment]
What it says when crashing? Any error logs or anything? Please provide us with something useful. Without this we can't help you

**Comment by SKaterTux** [comment]
There is literally nothing, just opens the app, freezes and the closes it.
I can't do anything else with it.

**Reply by KRTirtho** [reply]
Can you provide the `C:/Users/<user>/Documents/.spotube_logs` file?

**Comment by olivier2** [comment]
Hey @SKaterTux , I installed Windows 11 23H2 (Win11_23H2_English_x64v2.iso) in a VM to test this issue. Besides having to manually install the Visual C++ Redistributable 2019, I had no problem running [Spotube 3.4.1 built for Windows setup](https://github.com/KRTirtho/spotube/releases/download/v3.4.1/Spotube-windows-x86_64-setup.exe)

Can you try the latest Spotube version and let us know the outcome?
